main.py

```diff
@@ -49,17 +55,28 @@ def validate_id(cls, values):
 products = []
 
 
-# эдитпоин для добавления нового продукта
-@app.post('/products')
-async def add_product(product: Product):
-    try:
-        products.append(product.dict())
-        return {'status': 'success', 'data': product.dict()}, 201
-    except ValidationError as e:
-        return {'status': 'error', 'message': e.errors()}, 400
-
-
-# эдитпоинт для получения списка всех продуктов
 @app.get('/products/all')
 async def get_all_products():
     return {'status': 'success', 'data': products}, 200
+
+
+# Define the endpoint to add a new Product to MongoDB
+@app.post("/products/add")
+async def add_product_to_db(product: ProductCreate):
+    # Generate a new ObjectId for the product
+    product_id = str(ObjectId())
+
+    # Create a new document with the product data
+    product_data = {
+        '_id': product_id,
+        'name': product.name,
+        'price': product.price,
+        'type': product.type,
+        'region': product.region
+    }
+
+    # Insert the new product document into the MongoDB collection
+    mongo_db.products.insert_one(product_data)
+
+    # Return the new product as a dictionary
+    return {'status': 'success', 'data': product_data}
```

Comment:
Why do you need to return `status` to the user? Use has HTTP status code for this. Just return `product_data`. Also, you need to return `201` status code